alpha-blog/app/views/articles/_article.html.erb

```diff
@@ -10,7 +10,7 @@
             <h5 class="card-title"><%= link_to article.title, article_path(article), class: "text-success" %></h5>
             <p class="card-text"><%= truncate(article.description, length: 100) %></p>
             <%= link_to 'View', article_path(article), class: "btn btn-outline-success" %>
-            <% if logged_in? && article.user == current_user %>
+            <% if logged_in? && (article.user == current_user || current_user.admin?) %>
```

Comment:
Just had a thought here.
If `current_user` is not nil, it means it is `logged_in` right?
If thats the case, we don't need `logged_in?`